test/start_server_test.js

```diff
@@ -51,4 +52,19 @@ suite('startServer', () => {
       .end(done);
   });
 
+  test('unknown cmd parameter should not throw exception', (done) => {
+	var argv = process.argv;
```

Comment:
That wouldn't be enough, i need to modify process.argv, so that command-line-args  generates the exception, since it looks at process.argv by default.